fluent/test/primitives_test.js

```diff
@@ -67,55 +65,43 @@ suite('Primitives', function() {
     });
 
     test('can be used as a value', function(){
-      const msg = bundle.getMessage('foo');
-      const val = bundle.format(msg, args, errs);
+      const val = bundle.format('foo', args, errs);
       assert.equal(val, 'Foo');
       assert.equal(errs.length, 0);
     });
 
-    test('is detected to be non-complex', function(){
-      const msg = bundle.getMessage('foo');
-      assert.equal(typeof msg, 'string');
-    });
-
```

Comment:
It would probably be better to mark this tests as skipped with `test.skip(...)` rather than just removing it.